rskj-core/src/main/java/org/ethereum/core/TransactionBuilder.java

```diff
@@ -129,4 +129,18 @@ public Transaction build() {
     public TransactionBuilder data(String data) {
         return this.data(data == null ? null: Hex.decode(data));
     }
+    
+    public TransactionBuilder from(TransactionArguments args, byte chainId) {
```

Comment:
I like this one and I also think it should just return a `Transaction`, since it's a complex construction, and it also makes it harder to distinguish which ones are `Transaction` fields and which ones are complex constructions

Reply:
also can we rename it to something more expressive? (from what?) It could be something like 
```java
transacationBuilder.buildByTxArgsAndChainId(args, chainId)
```